rotkehlchen/db/db_cache.py

```diff
@@ -0,0 +1,74 @@
+from enum import auto
+from rotkehlchen.errors.serialization import DeserializationError
+from rotkehlchen.types import Timestamp
+from rotkehlchen.utils.mixins.enums import Enum
+
+
+DBCacheFieldTypes = int | Timestamp
+
+
+class DBCache(Enum):
+    """It contains all the values that can be stored in the `key_value_cache` table of the DB"""
+    LAST_WRITE_TS = auto()
```

Comment:
This is a very silly way to do this enum. You are adding A LOT of extra logic to "serialize/deserialize" and calling the "serialize()` function in every single DB write. This is a very very bad idea. Just use the value of the enum and change it away from auto() to the string value.

Reply:
you didn't do this one

Reply:
I believe you pushed this file instead of renamign